src/games/supported/BeamRider.hpp

```diff
@@ -68,6 +68,12 @@ class BeamRiderSettings : public RomSettings {
 
         virtual int lives() { return isTerminal() ? 0 : m_lives; }
 
+        // returns a list of difficulties that the game can be played in
+        // in this game, there are 2 available difficulties. The game's
+        // manual says there are three modes but at least from my ROM
+        // I couldn't find them. I found two difficulties.
+        DifficultyVect getAvailableDifficulties();

```

Comment:
I think the Atari 2600 only has two levels of difficulty: Advanced & Basic. Can you update the comment?

Reply:
I misread something in its manual. I removed the comment. However, it is still weird. The manual reads:

```
4. Select level of difficulty by pressing it's corresponding number on the
keypad.

level: Begins at sector:
1 1
2 5
3 10

You will immediately be in the Space Station. It's control panel
displays your score, two reserve ships, sector, number of enemy saucers
in sector and three torpedoes.
```

However, nor even in the Stella I was able to change modes (and it is not being supported in the previous PR). For now I'll leave it with 2 difficulties and zero modes.